gradoop-flink/src/main/java/org/gradoop/flink/model/api/tpgm/TemporalGraphOperators.java

```diff
@@ -42,6 +50,107 @@
   // Unary Operators
   //----------------------------------------------------------------------------
 
+  /**
+   * Extracts a snapshot of a temporal graph using a given temporal predicate. This will calculate
+   * the subgraph of a temporal graph induced by the predicate.
+   *
+   * The resulting graph is a new logical graph.
+   */
+  default TemporalGraph snapshot(TemporalPredicate predicate) {
+    return callForGraph(new Snapshot(Objects.requireNonNull(predicate)));
+  }
+
+  /**
+   * Extracts a snapshot of a temporal graph using the temporal predicate {@code AS OF t} where
+   * {@code t} is a timestamp in ms as long. This will calculate the subgraph of a temporal graph
+   * induced by the predicate.
+   *
+   * The resulting graph is a new logical graph.
```

Comment:
Clarify that the result is a temporal graph. (Again, should be clear by the return type of this method, but just to be sure.)